pinot-common/src/main/java/com/linkedin/pinot/common/utils/ZkStarter.java

```diff
@@ -137,31 +139,39 @@ public static ZookeeperInstance startLocalZkServer(final int port) {
    * @param dataDirPath The path for the Zk data directory
    */
   public synchronized static ZookeeperInstance startLocalZkServer(final int port, final String dataDirPath) {
+    // Disable SASL-authenticated for starting local zk server
+    String isSaslEnable = System.setProperty(ZK_SASL_CLIENT, "false");
```

Comment:
Why are we doing this?

Reply:
There's no need to enable SASL--authenticated if we start a local zk server. So I think there's no harm for us to reduce one possible cause for the timeout.